cls/_zpkg/isc/sc/git/Socket.cls

```diff
@@ -12,7 +12,8 @@ Property OriginalDevice;
 ClassMethod Run()
 {
      If %request.Get("method") = "preview" {
-         Do ##class(SourceControl.Git.API).Pull(1)
+		set branchName = ##class(SourceControl.Git.Utils).GetCurrentBranch()
+		do ##class(SourceControl.Git.Utils).RunGitWithArgs(.errStream, .outStream, "pull",  "origin/" _ branchName)
```

Comment:
Preview is not working correctly. Here we need to do a "git fetch", and then a "git diff". There should be no "git pull".